README.md

```diff
@@ -1,2 +1,21 @@
+---
+output: html_document
+---
 # hw01
 Repository for first CFSS homework assignment
+
+# Introduction
+I am a graduate student in the _MAPSS_ program with an economics concentration. 
+The list of classes that I am taking this quarter include:
+
+1. Firm Dynamics and Innovation
+2. Analysis of Micro Data
+3. This Class
+
+You can learn more about the MAPSS program through its website:
+https://mapss.uchicago.edu/
```

Comment:
Alternatively, you could write this `You can learn more about the MAPSS program through [its website](https://mapss.uchicago.edu/)`, which when rendered looks like

You can learn more about the MAPSS program through [its website](https://mapss.uchicago.edu/)